### DISCUSSION TITLE
Set the .env key GOOGLE_APPLICATION_CREDENTIALS to the JSON file path.

### DISCUSSION THREAD

**Discussion by stellar-security-os** [imdhemy/laravel-in-app-purchases#406] [Q&A]
Set the .env key GOOGLE_APPLICATION_CREDENTIALS to the JSON file path.

How should the path look like?

**Comment by imdhemy** [comment]
It should be a relative path from your base path. 

https://github.com/imdhemy/laravel-in-app-purchases/blob/bd3992938a62ae89a7d1a142d2de90c3aa035d81/config/liap.php#L44
